src/test/fixtures/utils.js

```diff
@@ -250,15 +250,13 @@ export function removeRootOverlayElement() {
  * Usage:
  * changeSelect({ from: 'Timing Data', to: 'Deallocations' });
  */
-export function createSelectChanger(renderResult: RenderResult<>) {
```

Comment:
Note: RenderResult isn't exported by the latest libdef. I _could_ add it, but then decided we could as well rewrite this function using `screen` instead of the render result, and this makes everything simpler.